autobahn/wamp_call_options.ipp

```diff
@@ -36,19 +36,44 @@ namespace autobahn {
 
 inline wamp_call_options::wamp_call_options()
     : m_timeout()
+    , m_disclose_me()
 {
 }
 
 inline const std::chrono::milliseconds& wamp_call_options::timeout() const
 {
-    return m_timeout;
+    return *m_timeout;
```

Comment:
In light of changes like this one, I wonder whether it would be better (more readable, less new methods, etc.) if `timeout()` and `disclose_me()` would just return `boost::optional` values themselves instead of hiding them behind an `is_*_set()` interface.

I guess this way is good for keeping source compatibility. I'm not sure how much value is put on that in this project vs. cleaner (imho) alternatives.